style.css

```diff
@@ -120,12 +125,221 @@
 .hamburger {
   text-decoration: none;
   list-style: absolute;
-  position: fixed;
-  padding-left: 300px;
-  left: 16.67%;
-  right: 16.67%;
+  position: absolute;
+  left: 68%;
   font-size: larger;
+  font-weight: bold;
   border: none;
   color: #6070ff;
   background-color: transparent;
 }
+
+.works {
+  display: grid;
+  grid-template-rows: auto auto auto auto;
+  gap: 50px;
+}
+
+.works-1 > div {
+  border: 1px solid black;
+}
+
+.work-card {
+  background-color: #fff;
+  border-radius: 24px;
+  border: 1px solid #dfe1e6;
+  padding-top: 10px;
+  margin-top: 20px;
+  margin-bottom: 42px;
+  width: 360px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-left: 13px;
+}
+
+.work-card-img {
+  object-fit: cover;
+  align-items: center;
+  justify-content: center;
+  margin-top: 12px;
+}
```

Comment:
- [ ] Kindly make sure your project image place holder is aligned exactly as how it looks in the provided Figma design. Please adjust the image width. :wink: 
 
   - ### Your design
   
  ![image](https://user-images.githubusercontent.com/87197412/185870643-675e5a73-5cc9-48d2-a9c5-a9d3b9272990.png)


   - ### Figma design
  
  ![image](https://user-images.githubusercontent.com/87197412/185870705-565c46f3-55d1-4e3f-8320-643d515861b3.png)